rules_py/ipython/WORKSPACE

```diff
@@ -0,0 +1,50 @@
+workspace(name = "rules_py_ipython_example")
```

Comment:
so far our convention in other Aspect rulesets has been that nested workspaces are under an e2e/ folder, and are just meant to protect us from bugs (minimal commenting and content)
User-facing examples are in the examples/ folder and rely on the root WORKSPACE file.

I know we've had a few different iterations on this. It's nice that you can just `bazel test ...` from the root here and cover all the examples

Reply:
Then write_source_files doesn't conform to this? 
https://github.com/aspect-build/bazel-examples/tree/main/write_source_files

> It's nice that you can just bazel test ... from the root here

"here" being this repo, or the rule set? As a lot of the examples in this repo have their own WORKSAPCE etc, so `bazel test ...` at the root of this repo would kinda do nothing.